.github/workflows/main.yml

```diff
@@ -0,0 +1,73 @@
+name: Node.js CI
+
+on: [push]
+
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+    timeout-minutes: 10
+    strategy:
+      matrix:
+        node-version: [14.x]
+    steps:
+      - uses: actions/checkout@v2
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v1
+        with:
+          node-version: ${{ matrix.node-version }}
+      - run: npm install
+      - name: Run eslint
+        run: npm run lint
+  test-node:
+    runs-on: ubuntu-latest
+    needs: [lint]
+    timeout-minutes: 10
+    strategy:
+      matrix:
+        node-version: [12.x, 14.x]
+    steps:
+      - uses: actions/checkout@v2
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v1
+        with:
+          node-version: ${{ matrix.node-version }}
+      - run: npm install
+      - name: Run test with Node.js ${{ matrix.node-version }}
+        run: npm run test-node
+#  test-karma:
```

Comment:
Are karma tests broken at the moment?

Reply:
@gannan08 - yeah, they are. This lib didn't have a karma suite before; I started to add it in this PR, but it still needs more work.